client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -312,30 +313,37 @@ export class SeoForm extends React.Component {
 				<QuerySiteSettings siteId={ siteId } />
 				{ siteId && <QueryJetpackPlugins siteIds={ [ siteId ] } /> }
 				{ siteIsJetpack && <QueryJetpackModules siteId={ siteId } /> }
-				{ ( isSitePrivate || isSiteHidden ) && hasSiteSeoFeature( selectedSite ) && (
-					<Notice
-						status="is-warning"
-						showDismiss={ false }
-						text={ ( function () {
-							if ( isSitePrivate ) {
-								if ( siteIsComingSoon ) {
+				{ ( isSitePrivate || isSiteHidden || siteIsComingSoonV2 ) &&
```

Comment:
Haven't tested yet, but I was thinking that maybe we don't need `siteIsComingSoonV2` here, since it's already covered by `isSiteHidden`.

If the user has a coming-soon-v2 site, and they've somehow manauged to change their `blog_public` setting to `1`, then there's no reason to prevent them from changing SEO settings. Google should happily index their coming-soon-v2 site.

Reply:
While coming soon v2 currently means site is hidden I didn't want to make the assumption since they're different options and have different text below.